forge-gui/res/adventure/common/maps/map/vampirecastle/wastetown..tmx

```diff
@@ -1,38 +1,183 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The file was originally named with an extra dot before the extension (..tmx) and it remains so right now as well, is that alright? Maybe can be fixed at the reference point (wherever it is) to avoid the extra dot, just need to make sure that it only has one both in the filename and in the references so that it loads correctly.

Reply:
This has been the case ever since I started contributing, it's generated automatically as far as I know

Reply:
Alright, let's keep it as is :)